WordFrequency.Tests/WordFrequencyGameTest.cs

```diff
@@ -60,7 +60,7 @@ private void Validate_Input_words_process_to_expected_word(string inputStr, stri
         {
             WordFrequencyGame game = new WordFrequencyGame();
             //When
-            string actualResult = game.GetResult(inputStr);
+            string actualResult = game.CalculateWordFrequency(inputStr);
```

Comment:
personally do not like change the name of entry method that may bring errors to outside consumers. could have more discussion about this...